circuit_knitting/forging/entanglement_forging_knitter.py

```diff
@@ -595,6 +633,14 @@ def _estimate_expvals(
     Returns:
         The expectation values for the tensor circuits and superposition circuits
     """
+    # Ignore HF energy calculation. We will hard-code it later.
```

Comment:
the changes to this function need to happen a level higher. we're not guaranteed the first ansatze is the HF bitstring since this is a function called on arbitrary batches of circuits